pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -90,6 +90,7 @@ def test_to_M8():
 #####
 # DateOffset Tests
 #####
+_ApplyCases = List[Tuple[int, Dict[datetime, datetime]]]
```

Comment:
```suggestion
_ApplyCases = List[Tuple[BaseOffset, Dict[datetime, datetime]]]
```

Kind of tricky but I don't think `int` is in play at all here but these are rather all `BaseOffset` subclasses.

This only "works" because there is no stub for `pandas.tseries.offsets` and all of the instances here resolve to `Any`. Assumedly if we stubbed that module it would cause failures if this stays as `int`

Reply:
good point. Have just pushed this change.